sdk/tables/azure-data-tables/azure/data/tables/aio/_table_client_async.py

```diff
@@ -627,13 +629,13 @@ async def get_entity(
     async def upsert_entity(
         self, entity: EntityType, mode: Union[str, UpdateMode] = UpdateMode.MERGE, **kwargs
     ) -> Mapping[str, Any]:
-        """Update/Merge or Insert entity into table.
+        """Updates/Merges or Inserts an entity into a table.
```

Comment:
Maybe something like:
```suggestion
        """Updates (merge or replace) or Inserts an entity into a table.
```

Reply:
Updated